SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -555,34 +570,47 @@ void DeleteGeneratedSchemaFiles()
 			UE_LOG(LogSpatialGDKSchemaGenerator, Error, TEXT("Could not clean the generated schema directory '%s'! Please make sure the directory and the files inside are writeable."), *SchemaOutputPath);
 		}
 	}
+}
+
+void CreateSchemaFolder()
+{
+	IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+	FString SchemaOutputPath = GetDefault<USpatialGDKEditorSettings>()->GetGeneratedSchemaOutputFolder();
 	PlatformFile.CreateDirectory(*SchemaOutputPath);
```

Comment:
Should probably use `CreateDirectoryTree` to ensure all parent folders exist.

Reply:
Can do that, yes.

Reply:
Changed to CreateDirectoryTree